setup/ci/gcb.md

```diff
@@ -73,3 +77,46 @@ if the build produces a Docker image, you can add an expected artifact of type *
 `gcr.io/my-project-id/my-application` (replacing `my-project-id` and `my-application` with
 appropriate values). You can then [use the produced image](/reference/artifacts/in-pipelines/)
 in downstream stages.
+
+## Configure a Google Cloud Build Stage
```

Comment:
I think this should be in /guides/pipeline/ rather than in setup. 
Everything from here on down.

Reply:
I was wondering if that might be better for the stage instructions, but it looked like that section had mostly quick summaries rather than detailed instructions.  Let's chat about where this should live.